climada/util/coordinates.py

```diff
@@ -518,6 +519,29 @@ def dist_to_coast(coord_lat, lon=None, signed=False):
         dist[coord_on_land(geom.geometry.y, geom.geometry.x)] *= -1
     return dist
 
+def get_dist_to_coast_nasa_tif():
```

Comment:
maybe a `util.datasets` module would be useful?
(though we'd also need `Hazard.datasets`, `Exposures.datasets`, ...)

Reply:
Why would one need a `Hazard.datasets` and `Exposures.datasets` ? These are present in the config file right?

Reply:
I now transformed this into an non-official function so that the official API is not cluttered:
https://github.com/CLIMADA-project/climada_python/blob/b74a06446e153b36dea8ab5f948002858160b168/climada/util/coordinates.py#L522-L526
If we want to introduce a function like that `get_external_data` at a later stage, we can savely change or remove this one without worrying about backwards compatibility.